node/src/command.rs

```diff
@@ -1,11 +1,9 @@
 use crate::{
     chain_spec,
-    cli::{
-        Cli,
-        RelayChainCli,
-        Subcommand,
-    },
+    cli::{Cli, RelayChainCli, Subcommand},
+    service::{new_partial, DataHighwayRuntimeExecutor}
 };
+use datahighway_runtime::RuntimeApi;
 use codec::Encode;
 use cumulus_client_service::genesis::generate_genesis_block;
 use cumulus_primitives_core::ParaId;
```

Comment:
the function `fn load_spec` in this file may need to be updated based on changes in chain_spec.rs

Reply:
plaese change the value returned from support_url function to `https://github.com/DataHighway-DHX/node/issues/new`

Reply:
please change value returned from copyright_start_year to `2021`